docs/maps/vector-style.asciidoc

```diff
@@ -51,14 +51,11 @@ The sample web logs `bytes` field ranges from 0 to 18,000. This is the domain ra
 The smallest feature has a symbol radius of 1, and the largest feature has a symbol radius of 24. This is the style range.
 The `bytes` property value for each feature will fit on a linear scale from the range of 0 to 18,000 to the style range of 1 to 24.
 
-To ensure symbols are consistent as you pan, zoom, and filter the map, quantitative data driven styling uses {ref}/search-aggregations-metrics-extendedstats-aggregation.html[extended_stats aggregation] to retrieve statistical metadata. Extended_stats is not available for numeric property values from the <<maps-aggregations, metric aggregations>> count, sum, and unique count.
+For color styles, values are fit from the domain range to the color ramp with one of the following:
 
-To configure extended_stats, click the gear icon image:maps/images/gear_icon.png[]. Set *Sigma* to a smaller value to minimize outliers by moving the range minimum and maximum closer to the average. Clear the *Calculate range from indices* checkbox to turn off the extended_stats aggregation request. The gear icon is not available for numeric property values from the <<maps-aggregations, metric aggregations>> count, sum, and unique count.
-
-NOTE: When extended_stats is not used, symbols might be inconsistent as users pan, zoom, and filter the map. Without extended_stats, the range is calculated with data from the local layer. The range is re-calculated when layer data changes.
-
-[role="screenshot"]
-image::maps/images/extended_stats_config.png[]
+* *Interpolate (default)*. Color ramp is divided into 8 bands. Values are interpolated between min and max to a color band on a linear scale.
```

Comment:
Make this bullet point consistent with the others--start with a verb. Maybe something like:

Interpolate values between min and max to a color band on a linear scale.  The color ramp is divided into eight bands.